app-backend/export/src/main/scala/com/azavea/rf/export/model/ExportDefinition.scala

```diff
@@ -0,0 +1,20 @@
+package com.azavea.rf.export.model
+
+import spray.json.DefaultJsonProtocol._
```

Comment:
We've switched to using `circe` in the API -- does it make sense to be consistent across all subprojects?